docs/source/making-plugins.rst

```diff
@@ -343,9 +343,9 @@ no entrypoints, it will issue a ``FutureWarning``.
 Python API to Driver Discovery
 ''''''''''''''''''''''''''''''
 
-.. autofunction:: intake.source.discovery.autodiscover
-.. autofunction:: intake.source.discovery.enable
-.. autofunction:: intake.source.discovery.disable
+.. autofunction:: intake.source.discovery.DriverSouces.register_driver
```

Comment:
intake.source.discover.drivers.register_driver (`drivers` is an import-time instance of DriverSources)